website/addons/base/serializer.py

```diff
@@ -56,7 +60,32 @@ def serialized_user_settings(self):
 
 class OAuthAddonSerializer(AddonSerializer):
 
-    REQUIRED_URLS = ['importAuth', 'folders', 'config', 'deauthorize', 'accounts']
+    @property
+    def credentials_owner(self):
+        return self.user_settings.owner if self.user_settings else None
+
+    @property
+    def user_is_owner(self):
+        if self.user_settings is None or self.node_settings is None:
+            return False
+
+        user_accounts = self.user_settings.external_accounts
+        return bool(
+            (
+                self.node_settings.has_auth and
+                (self.node_settings.external_account in user_accounts)
+            ) or len(user_accounts)
+        )
+
+    @property
+    def serialized_urls(self):
+        ret = self.addon_serialized_urls
+        # Make sure developer returns set of needed urls
+        for url in self.REQUIRED_URLS:
```

Comment:
Probably not. `OAuthAddonSerializer` should never be instantiated, and anything that inherits directly from it rather than from a subclass (like `StorageAddonSerializer`) should be defining its own `REQUIRED_URLS`, like Dataverse, because it's probably not generalizable enough.